### DISCUSSION TITLE
"This browser doesn't seems to have Server Sent Events support."

### DISCUSSION THREAD

**Discussion by paulbruzz** [hoppscotch/hoppscotch#3327] [Q&A]
Hello,

I have been trying to send real-time requests to a locally-hosted **.NET Framework** API, which uses the **SignalR** library, by using the SSE protocol.

However, every time I click "Start" to submit the request, Hoppscotch immediately disconnects and returns the following response body: **"This browser doesn't seems to have Server Sent Events support.".**

I am experiencing this problem on Edge (v116), Chrome (v116) and Firefox (v117), all of which are updated to their current version (which, according to [this table](https://caniuse.com/eventsource), should actually support SSE).

I have installed and enabled the browser extension as explained [here](https://github.com/hoppscotch/hoppscotch/discussions/2051) and set the interceptor to "Browser extension: v0.25" on all three browsers, but this has not fixed the problem.

Could you please suggest a solution or a workaround?

Thank you in advance.

**Comment by KaKi87** [comment]
I'm experiencing the same problem even though `EventSource` itself works fine in browser, and even the desktop app throws this error...

**Comment by Kolterdyx** [comment]
This is also happening to me in the latest version (`25.9.0` at time of writing)

**Comment by johnhenry** [comment]
I'm experience this on the installed desktop appllication.

**Comment by park-jihoo** [comment]
I'm also experiencing with latest desktop version.

**Comment by shinolex** [comment]
Have the same issue. Just tried out and Hoppscotch says "This browser doesn't seems to have Server Sent Events support." even tho it does. Any news about this issue?
